### PR TITLE
[REF-5833] move mlops/mlcomp under mlpiper

### PR DESCRIPTION
**Pull request by yakov-g**

### CODE REVIEW

**Comment by liororama** [comment]
Make sure build is passing....

File: mlcomp/tests/test_mlcomp_python.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 import parallelm.pipeline.json_fields as json_fields
 
+REFLEXCOMMON_PATH = "../../reflex-common"
```
Comment (kedarw) [inline mlcomp/tests/test_mlcomp_python.py]: Move this to constants file within this repo

File: mlcomp/tests/test_mlpiper_cmdline.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import subprocess
 import sys
 
+COMPONENTS_PATH = "../../reflex-algos/components/Python"
```
Comment (kedarw) [inline mlcomp/tests/test_mlpiper_cmdline.py]: same as above

**Review by kedarw — APPROVED** [review]
Take care of 2 comments and build.

**Review by zohar-mizrahi — APPROVED** [review]

File: mlcomp/tests/test_mlcomp_python.py (hunk 1 above)
Comment (yakov-g) [reply mlcomp/tests/test_mlcomp_python.py]: This is related to tests only, not to the mlcomp package.
Probably I'm not going to move it anythere fo now.

**Pull request merged**